src/lib/storage/storage_manager.cpp

```diff
@@ -1,21 +1,37 @@
 #include "storage_manager.hpp"
 
+#include <sys/fcntl.h>
+#include <sys/mman.h>
 #include <memory>
 #include <string>
 #include <utility>
 #include <vector>
 
 #include "hyrise.hpp"
+#include "import_export/binary/binary_writer.hpp"
 #include "import_export/file_type.hpp"
-#include "logical_query_plan/abstract_lqp_node.hpp"
 #include "operators/export.hpp"
 #include "operators/table_wrapper.hpp"
 #include "scheduler/job_task.hpp"
 #include "statistics/generate_pruning_statistics.hpp"
 #include "statistics/table_statistics.hpp"
+#include "storage/base_dictionary_segment.hpp"
+#include "storage/dictionary_segment.hpp"
+#include "storage/dictionary_segment/dictionary_segment_iterable.hpp"
+#include "storage/value_segment.hpp"
 #include "utils/assert.hpp"
 #include "utils/meta_table_manager.hpp"
 
+namespace {
```

Comment:
why are we using an anonymous namespace here?

Reply:
Removed in de4795d.